### PR TITLE
Page with greeting header

### PR DESCRIPTION
**Pull request by pmpeirano2**
added the h1 header displaying the welcome message.

### CODE REVIEW

**Review by pmpeirano2 — COMMENTED** [review]
This is good but could you add a message. "we are ready to serve you" and make the colour red

**Review by pmpeirano — DISMISSED** [review]
this is good but could you add a message "we are ready to serve you" and make it red

**Review by pmpeirano2 — COMMENTED** [review]
yes, i will add that right away

**Review by pmpeirano — CHANGES_REQUESTED** [review]
could you change the message

File: index.html
Comment (pmpeirano) [inline index.html:1]: add code here pls

**Review by pmpeirano — COMMENTED** [review]
change the code there plz

**Review by pmpeirano — APPROVED** [review]
Awesome! Thank you.

**Pull request merged**